apiserver/keymanager/keymanager.go

```diff
@@ -194,8 +216,9 @@ func (api *KeyManagerAPI) AddKeys(arg params.ModifyUserSSHKeys) (params.ErrorRes
 		return result, nil
 	}
 
-	if !api.canWrite(arg.User) {
-		return params.ErrorResults{}, common.ServerError(common.ErrPerm)
+	// NOTE: arg.User isn't a tag, but a username.
```

Comment:
The comment is necessary when passing in entity.Tag (which is awful, we should not have used Entities for passing in non-tagged things). It's just confusing here.